swagger_parser/test/e2e/e2e_test.dart

```diff
@@ -198,6 +198,19 @@ void main() {
       );
     });
 
+    test('discriminated_one_of.3.0_mappable', () async {

```

Comment:
It would be nice to combine this test with the `discriminated_one_of.3.0` test above.

I've opened #291 to suggest some changes to the test structure to allow this.